PracticeCode/AntProjects/dividerFunction1_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from dividerFunction1 import divider
+
+# Test for dividerFunction1 pycode.
+# Test 1; func will throw an error when dividing by zero
+# Test 2; func will return a float when dividing by an Int by a float
+
+
+#divider(3,1)
+
+class TestDividerFunc(unittest.TestCase):
+    def test_zero_division_error(self):
+        # Make sure value errors is raised when int is divided by zero.
+        self.assertRaises(ZeroDivisionError, divider, 3, 0)
+
+    def test_float_result(self):
+        # capture result of divider func
+        result = divider(3, 1.0)
+        # expected result of func
+        expected_result = 3.0
+        self.assertEqual(result, expected_result)
+
+
```

Comment:
Add more tests.
- what happens when number2 > number1?
- what happens when number1 == number2?
- what happens when they're both floats?
- what happens when they're both Ints?
You don't need to check the type. You can simply check the return type like above

Reply:
Added new funcs to match request above.